### ISSUE TITLE
Remove default language

### ISSUE THREAD

**Issue by RoelantStegmann** [agentejo/cockpit#820]
Could it be an option to remove the default from the languages? Its a bit tiring when having required fields, having to fill everything in another time (which isn't used in my application).

If desired, I could try to help with coding :)

**Comment by mpartipilo** [comment]
Another alternative could be copying the value entered in the default
language field over to the required fields in other languages.

**Comment by RoelantStegmann** [comment]
True. Another option could be to specify the 'default' (e.g. say that the default is 'en').

**Comment by mpartipilo** [comment]
Now that you mention it, I had not read your first post correctly. I assumed the first language in the list of languages would be picked up as default and found out that it's not the case.

**Comment by DigitalGoldfish** [comment]
I am not sure I get what you are trying to achieve. I use the default language if there is only one language. If there are two I only add the non-native language to the languages array (usually 'en') and treat the default language as "de". 

Can your problem be solved by simply changing the label of the "default"  language?

**Comment by mpartipilo** [comment]
Maybe, as long as it’s also returned by the api when querying the language
list.

**Comment by RoelantStegmann** [comment]
Yes, think just renaming it would also be a work around. I prefer to explicitly query rather than assuming the default is some language :)

**Comment by raffaelj** [comment]
If you add `i18n: de` to `config.yaml`, you can query the default language with your language tag directly.

```yaml
i18n: de

languages:
  en: English
```
